domains/saad.json

```
@@ -4,6 +4,7 @@
     "email": "github.com.haunt330@passinbox.com"
   },
   "records": {
-    "CNAME": "hyodu.deno.dev"
+    "A": "34.120.54.55",
+    "AAAA": "2600:1901:0:6d85::"
```

A and AAAA records should be in an array
```suggestion
    "A": ["34.120.54.55"],
    "AAAA": ["2600:1901:0:6d85::"]
```